static/src/javascripts/projects/commercial/modules/dfp/define-slot.js

```diff
@@ -111,7 +97,7 @@ const defineSlot = (adSlotNode: Element, sizes: Object): Object => {
         if (isEligibleForOutstream(slotTarget)) {
             allowSafeFrameToExpand(slot);
         }
-        slotReady = setHighMerchSlotTargeting(slot, slotTarget);
+        slotReady = Promise.resolve();
```

Comment:
Amazing! Just having a quick look I notice `slotReady` is now always set to `Promise.resolve()`. Potentially this variable is no longer needed at all? And where it was used we can directly invoke the success callback

Reply:
Just seen this, let me check :)

Reply:
You are right @ioanna0 I will let you do that one in a separate PR :)

Reply:
Done in https://github.com/guardian/frontend/pull/22553